src/bundle/Resources/public/js/scripts/admin.content.tree.js

```diff
@@ -20,7 +32,6 @@
         );
     }
 
-    doc.body.addEventListener('ibexa-tb-rendered:ibexa-content-tree', removeContentTreeContainerWidth);
-
+    addContentTreeListeners();
```

Comment:
Maybe we could do an early return before invoking those functions? We would not have to add guards in every function in the future.

Reply:
I thought about this, but I'm not sure if adding return in main body of file is good and clean idea. In fact, the only reason it would work here is because we wrap everything in IIFE

What does other thinks? @lucasOsti  @tischsoic